Library/Homebrew/brew.rb

```diff
@@ -88,7 +88,7 @@ def require?(path)
   end
 
   if internal_cmd
-    Homebrew.send cmd.to_s.tr("-", "_").downcase
+    Homebrew::InstallCommand.send cmd.to_s.tr("-", "_").downcase
```

Comment:
I'm not entirely sure I understand the problem, but I _think_ you might want to try checking if `InstallCommand` has a method for the command, and falling back the old behaviour if not:

```ruby
if Homebrew::InstallCommand.respond_to?(…)
  …
else
  …
end
```